api/build.gradle.kts

```diff
@@ -28,3 +28,9 @@ blossom {
 tasks.javadoc {
     enabled = project.hasProperty("enable-javadoc")
 }
+
+java {
```

Comment:
What's your purpose for this? Everything compiles fine using a Java 17 compiler targeting Java 8, we don't need to use Java 8 for the API like this.